src/Components/CreateImageWizard/steps/Repositories/Repositories.tsx

```diff
@@ -389,207 +404,404 @@ const Repositories = () => {
+                            onSelect: (_, isSelecting) =>
+                              handleAddRemove(repo, isSelecting),
+                            isDisabled: isDisabled,
+                          }}
+                          title={disabledReason}
                         />
-                      </Td>
-                    </Tr>
-                  );
-                })}
-              </Tbody>
-            </Table>
-          )}
-        </PanelMain>
-      </Panel>
-      <Pagination
-        itemCount={count ?? PAGINATION_COUNT}
-        perPage={perPage}
-        page={page}
-        onSetPage={(_, newPage) => setPage(newPage)}
-        onPerPageSelect={handlePerPageSelect}
-        variant={PaginationVariant.bottom}
-      />
-    </Grid>
-  );
+                        <Td dataLabel={'Name'}>
+                          {name}
+                          {origin === ContentOrigin.UPLOAD ? (
+                            <UploadRepositoryLabel />
+                          ) : (
+                            <>
+                              <br />
+                              <Button
+                                component="a"
+                                target="_blank"
+                                variant="link"
+                                icon={<ExternalLinkAltIcon />}
+                                iconPosition="right"
+                                isInline
+                                href={url}
+                              >
+                                {url}
+                              </Button>
+                            </>
+                          )}
+                        </Td>
+                        <Td dataLabel={'Architecture'}>
+                          {distribution_arch || '-'}
+                        </Td>
+                        <Td dataLabel={'Version'}>
+                          {distribution_versions || '-'}
+                        </Td>
+                        <Td dataLabel={'Packages'}>{package_count || '-'}</Td>
+                        <Td dataLabel={'Status'}>
+                          <RepositoriesStatus
+                            repoStatus={status || 'Unavailable'}
+                            repoUrl={url}
+                            repoIntrospections={last_introspection_time}
+                            repoFailCount={failed_introspections_count}
+                          />
+                        </Td>
+                      </Tr>
+                    );
+                  })}
+                </Tbody>
+              </Table>
+            )}
+          </PanelMain>
+        </Panel>
+        <Pagination
+          itemCount={count ?? PAGINATION_COUNT}
+          perPage={perPage}
+          page={page}
+          onSetPage={(_, newPage) => setPage(newPage)}
+          onPerPageSelect={handlePerPageSelect}
+          variant={PaginationVariant.bottom}
+        />
+      </Grid>
+    );
+  } else {
```

Comment:
I think "On the repositories step, only the repositories in the template should be shown, and should be selected and read-only" already answers my question, but to double check for a peace of mind :sweat_smile:  - it's either "normal" repositories table or a template repos table rendered/ There's no valid scenario where we would need to render both at the same time, right?

Reply:
yep it's either or :) users shouldn't be able to add additional repositories if they're using a template